client/me/developer/features/use-features-list.tsx

```diff
@@ -0,0 +1,104 @@
+import { localizeUrl } from '@automattic/i18n-utils';
+import { useTranslate } from 'i18n-calypso';
+
+export const useFeaturesList = () => {
+	const translate = useTranslate();
+
+	return [
+		{
+			title: translate( 'SFTP, SSH, and WP-CLI', {
+				comment: 'Feature title',
+			} ),
+			description: translate(
+				'Run WP-CLI commands, automate repetitive tasks, and troubleshoot your custom code with the tools you already use.',
+				{
+					comment: 'Feature description',
+				}
+			),
+			linkLearnMore: localizeUrl( '/support/connect-to-ssh-on-wordpress-com' ),
```

Comment:
@Aurorum jsut to confirm - are we missing something or you missed `localizeUrl `?

Reply:
I might be missing something, but this doesn't work for me!

![ScreenRecording2024-01-24at13 39 53-ezgif com-video-to-gif-converter](https://github.com/Automattic/wp-calypso/assets/43215253/cc009272-2163-4263-8810-2e0c43c68853)

Reply:
Thanks for explanation. It doesn't work locally, but if you replace `localhost` with `wordpress.com` it will be ok. So it production it will work.

Reply:
Ah, I see, thanks for explaining! Will it still work for folks on https://horizon.wordpress.com/ or https://wpcalypso.wordpress.com/ though?

Perhaps not necessary, but I feel a feature like this might be disproportionately used by folks who use Horizon or WPCalypso.